Source/HAConnection.swift

```diff
@@ -180,4 +180,9 @@ public protocol HAConnection: AnyObject {
         initiated: @escaping SubscriptionInitiatedHandler,
         handler: @escaping (HACancellable, T) -> Void
     ) -> HACancellable
+
+    /// Write data to websocket connection
+    /// - Parameters:
+    ///   - request: The data request containing data to be written
+    func write(_ dataRequest: HARequest)
```

Comment:
I think it should be possible to "send" without knowing the request is a data request. Does that make sense? Like I'm saying "send an STT request, here's the payload, let me know when you've finished writing it" -- we don't expect a response, but we in theory do care about _that_ part.